docs/introduction/QuickStart.md

```diff
@@ -0,0 +1,200 @@
+---
+id: quick-start
+title: Quick Start
+sidebar_label: Quick Start
+hide_title: true
+---
+
+# Quick Start
+
+To bootstrap your next Redux project, here are some minimal examples to get you started (_copy -> paste -> tweak_).
+
+## Installation
+
+All of our examples will be using the official Redux package and the **[Redux Toolkit](https://redux-toolkit.js.org/)** addon package, so let's go ahead and install those.
+
+```bash
+# NPM
+npm install --save redux @reduxjs/toolkit
+
+# Yarn
+yarn add redux @reduxjs/toolkit
+```
+
+## The Redux Code
+
+Let's say we're building a _simple_ calculator. The calculator will be able to add or subtract numbers based on the user's input.
+
+First off, let's start by creating the actions necessary for this calculator to work. We'll need to be able to add and substract a value.
+
+### The Actions
+
+```js
+const add = createAction('ADD')
+const substract = createAction('SUBSTRACT')
+```
+
+Here we're using Redux Toolkit's `createAction` to create the necessary actions we'll be using for the app.
+
+Cool, now let's move onto the reducer.
+
+### The Reducer
+
+```js
+const calculator = createReducer(0, {
+  [add]: (state, action) => state + action.payload,
+  [subtract]: (state, action) => state - action.payload
+})
+```
+
+At first glance this may be a little strange to look at, but let's unpack it:
+
+- The first argument passed into `createReducer` is the initial state, which we'll start at `0`.
+- The second argument defines a "lookup table" object, where the key is the action type and the value is the reducer function associated with the action.
+- In case you aren't familiar with the weird `[add]` syntax, it is known as the [ES6 object "computed property" syntax](https://javascript.info/object#computed-properties). This allows us to create keys from actions made with `createAction` with ease (since the computed properties ultimately calls `toString()` on those variables, which `createAction` objects overrides with the value of its type property).
+
+The reducers' implementations should be fairly straightforward, simply adding or subtracting the action payload.
```

Comment:
This is making an unfair assumption on the reader's knowledge, consider revising the wording here.